rero_ils/modules/documents/jsonschemas/documents/document_issuance-v0.0.1.json

```diff
@@ -189,7 +189,7 @@
         }
       },
       {
-        "title": "Integrating resource",
+        "title": "rdami:1004",
```

Comment:
I understand why but looks like ugly as the JSONSchema is also a data model documentation.